modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -116,9 +116,9 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo SpriteTextureInfo = {
         "File",
         "Point Sprite Texture File",
-        "The path to the texture that should be used as the point sprite. Note that if "
-        "multiple textures option is set in the asset, by providing a texture folder, "
-        "this value will be ignored.",
+        "A path to a texture to use as the point sprite. Note that if multiple textures "
```

Comment:
```suggestion
        "The path to the texture to use as the point sprite. Note that if multiple textures "
```

I think the previous version was a bit easier to read since we are talking about a specific path to a specific texture